iosTests/ios/Podfile

```diff
@@ -13,15 +14,45 @@ end
 target 'SalesforceReactTests' do
   source 'https://cdn.cocoapods.org/'
   config = use_native_modules!
-  use_react_native!(:path => config["reactNativePath"])
+
+  # Flags change depending on the env values.
+  flags = get_default_flags()
+
+  use_react_native!(
+    :path => config[:reactNativePath],
+    :hermes_enabled => false,
+    :fabric_enabled => flags[:fabric_enabled],
+    :flipper_configuration => FlipperConfiguration.disabled,
+    :app_path => "#{Pod::Config.instance.installation_root}/.."
+  )
   pod 'React-RCTTest', :path => '../RCTTest'
   use_mobile_sdk!(:path => '../mobile_sdk/SalesforceMobileSDK-iOS')
-  pod 'SalesforceReact', :path => '../..'
+  pod 'SalesforceReact', :path => '../node_modules/react-native-force'
+end
+
+# To avoid Xcode 12 compilation errors in RNScreens and RNCMaskedView
```

Comment:
Xcode 12?  Should that be 13?

Reply:
It started in Xcode 12.

Reply:
That code is not new. I had it in the templates Podfile, I just made sure all the Podfile (tests / templates) are similar. We probably don't need it for the tests.